tests/e2e/cloudprovider/test-lb-service.sh

```diff
@@ -290,6 +291,11 @@ function test_forwarded {
     local public_ip=$(curl -sS ifconfig.me)
     local local_ip=$(ip route get 8.8.8.8 | head -1 | awk '{print $7}')
 
+    if [[ ${OCTAVIA_PROVIDER} == "ovn" ]]; then
```

Comment:
is this skipped because of service test which has x-forwarded-for annotation or what is the reason behind skipping the service test?

Reply:
OVN Octavia Provider is an L4 load balancer meaning that it has no ability to inject headers into the responses. See [^1] for information that OVN only supports TCP, UDP, SCTP and no HTTP as a protocol.

[^1] https://docs.openstack.org/ovn-octavia-provider/latest/admin/driver.html#limitations-of-the-ovn-provider-driver